Added options to monit GPU & SMART with variables (#34)
ansible-collections/community.beszel: roles/agent/README.md

Extra filesystems to be monitored by the Beszel binary agent. Configures the [EXTRA_FILESYSTEMS](https://beszel.dev/guide/additional-disks#binary-agent) environment variable in the agent systemd unit file.

```yaml
agent_smart_disks: []

> # Example with disks to be SMART monitored by Beszel

Reviewer: ```suggestion
# Example with disks to be SMART monitored by the Beszel binary agent
```

agent_smart_disks:
  - /dev/sda
  - /dev/nvme0n1
```

SMART devices to be monitored by the Beszel binary agent. Configures the [Capabilities, smartmontools and DeviceAllow](https://beszel.dev/guide/smart-data) in the agent systemd unit file.  
> [!IMPORTANT]

> > If NVME drives aren't showing up, please follow [this troubleshooting in Beszel documentation](https://beszel.dev/guide/smart-data#adjust-nvme-device-permissions)

Reviewer: ```suggestion
SMART devices to be monitored by the Beszel binary agent. Configures the [Capabilities, smartmontools and DeviceAllow](https://beszel.dev/guide/smart-data) in the agent systemd unit file.  

> [!IMPORTANT]
> If NVME drives do not show up, please follow the [troubleshooting](https://beszel.dev/guide/smart-data#adjust-nvme-device-permissions) documentation.
```

```yaml
agent_smart_interval: 1h
```

Interval between SMART checks by the Beszel binary agent. 

```yaml
agent_gpus: []
# Example for Nvidia GPU
agent_gpus:
  - path: /dev/nvidia0

>     type: nvidia0

Reviewer: ```suggestion
    type: nvidia
```
